app/assets/stylesheets/legacy/application/publishers.sass

```diff
@@ -303,7 +303,7 @@ body[data-controller="publishers"]
     .container
       .col-details
         margin-top: 30px
-      h4
+      h4.balance-header
```

Comment:
This has been driving me nuts. The `h4` style here I believe is only for the balance header, but it leaks out to all publisher pages. The long-term fix is migrating this page to bootstrap 4, this is a cheap hack.